app/views/payments/_payment.html.haml

```diff
@@ -0,0 +1,35 @@
+.div
+  .col-md-12.table-responsive
+    %table.table.table-hover
+      %thead
+        %tr
+          %th Ticket
+          %th Quantity
+          %th Price
+          %th Total
+      %tbody
+      - @unpaid_ticket_purchases.each do |ticket|
+        %tr
+          %td
+            = ticket.title
+          %td
+            = ticket.quantity
+          %td
+            = humanized_money_with_symbol ticket.price
+          %td
+            = humanized_money_with_symbol ticket.quantity * ticket.price
+
+= form_tag conference_payments_path do
+  %ul.price-tags.pull-right
+    %li
+      %a
+        = humanized_money_with_symbol @total_amount_to_pay
+  %script.stripe-button{ src: "https://checkout.stripe.com/checkout.js",
+        data: { amount: @total_amount_to_pay.cents,
+          email: current_user.email,
+          currency: @total_amount_to_pay.currency,
+          name: ENV['OSEM_NAME'] || 'OSEM',
+          description: "book your tickets",
+          key: Rails.application.secrets.stripe_publishable_key,
+          locale: "auto"}}
+  = link_to 'Edit Purchase', conference_tickets_path(@conference.short_title), class: 'btn btn-primary'
```

Comment:
I'm not too excited about how this page looks. The "Pay with card" button looks nothing like the other buttons in the rest of OSEM, so it looks out of place. Plus we have two blue buttons. And there's the green ribbon which looks a bit out of place.. I know we are not designers 😂 but maybe we can improve it a bit. An idea:

**Before**
<img width="718" alt="screen shot 2016-08-09 at 6 42 46 pm" src="https://cloud.githubusercontent.com/assets/2944/17525083/8d51fc9a-5e61-11e6-9d1f-1e5b0260435a.png">
 

**After**
<img width="699" alt="screen shot 2016-08-09 at 6 42 29 pm" src="https://cloud.githubusercontent.com/assets/2944/17525103/9c264da2-5e61-11e6-9706-dc1aa63bcf18.png">